projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-themes-active-endpoint.php

```diff
@@ -46,6 +46,11 @@ protected function switch_theme() {
 			return new WP_Error( 'theme_not_found', __( 'Theme is empty.', 'jetpack' ), 404 );
 		}
 
+		/**
+		* Trigger action before the switch theme happens.
+		*/
```

Comment:
Let's extend that docblock, so the script generating Codex pages from our codebase can pick up that new hook:

```suggestion
		 * Trigger action before the switch theme happens.
		 *
		 * @module json-api
		 *
		 * @since $$next-version$$
		 *
		 * @param string $theme_slug Directory name for the theme.
		 * @param mixed  $args       POST body data, including info about the theme we must switch to.
		 */
```